chore: release notes for hotfix 6.3.1 (#12788)
OpenTrons/opentrons-api: app-shell/build/release-notes.md

---

## Opentrons App Changes in 6.3.1

Welcome to the v6.3.1 release of the Opentrons App!

> There are no changes to the Opentrons App in v6.3.1, but please update to receive an update for your OT-2 that fixes some issues.

Reviewer: ```suggestion
There are no changes to the Opentrons App in v6.3.1, but it is required for updating the OT-2 software to fix some issues.
```
I'm not in love with this wording but as long as we don't say `please update…to update` then :shipit:
